src/js/helpers/awardSummaryHelper.js

```diff
@@ -34,33 +34,59 @@ export const isContract = (awardType) => [
     'definitive contract'
 ].includes(awardType);
 
+// takes core location object
+export const isUSAAward = (placeOfPerformance) => {
+    const countryCode = placeOfPerformance._countryCode;
+    const countryName = placeOfPerformance.countryName;
+    if (
+        countryCode === 'USA'
+        || countryCode === 'UNITED STATES'
+        || countryName === 'UNITED STATES'
+    ) return true;
+    return false;
+};
+
 // award overview recipient section - determines text and address to display to user
 // data can be found in
 export const getAwardTypeByRecordtypeCountyAndState = (
-    isFinancialAssistance,
+    awardType,
     placeOfPerformance,
     recordType
 ) => {
-    if (isFinancialAssistance) {
+    const countyCode = placeOfPerformance._countyCode;
+    const isUSA = isUSAAward(placeOfPerformance);
+    if (isAwardFinancialAssistance(awardType)) {
         // redacted due to PII
-        if (recordType === 3) return 'redactedDueToPII';
+        if (recordType === 3) {
+            if (!isUSA) {
```

Comment:
Not requesting changes here but expressing my personal preference here for conditions evaluated that are true. 

I think it's easeir to say "if x is true" rather than if x is false because then you have to ask the question "is it true that x is false" which is slightly confusing if you think about it too hard.